scripts/validator/index.ts

```diff
@@ -128,23 +133,32 @@ class Validator {
    */
   public async setCommission() {
     console.log(`\nSetting reward commission`);
+    const commission = +process.env.REWARD_COMMISSION * +process.env.REWARD_COMMISSION_MULTIPLIER;
```

Comment:
What is the Multiplier. What does it mean? Why do we need it? It should be clear from variable name.